babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateTable.java

```diff
@@ -50,18 +51,21 @@
 
   /** Creates a SqlCreateTable. */
   public SqlCreateTable(SqlParserPos pos, boolean replace, SetType setType, Volatility volatility,
```

Comment:
Let's keep the existing constructor and have it pass through a null tableAttributes list to the new constructor.

This allows existing callers (including ones outside of the Calcite codebase) to continue functioning even after this change.

Reply:
Committed a change for this